sdk/internal/perf/utils.go

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package perf
+
+import "fmt"
+
+// Helper function to sum a slice of integers
+func sumInts(ints []int) int {
+	var ret int
+	for _, i := range ints {
+		ret += i
+	}
+	return ret
+}
+
+// convert an integer to a string, left padding with zeros
+func leftPad(i int) string {
+	if i >= 100 {
+		return fmt.Sprintf("%d", i)
+	} else if i >= 10 {
+		return fmt.Sprintf("0%d", i)
+	} else if i > 0 {
+		return fmt.Sprintf("00%d", i)
+	}
+	return "000"
+}
+
+// Convert an integer to a string with comma separation in the format 1234 -> 1,234
+func commaIze(i int) string {
```

Comment:
You might be able to use the `message` package instead of writing this yourself:

https://pkg.go.dev/golang.org/x/text/message?utm_source=godoc

Reply:
Using this package, but this also forces users of client libraries to download it because every package uses `internal`. Something I don't know how we tackle yet.